app/services/item_service.rb

```diff
@@ -0,0 +1,15 @@
+class ItemService
+  def self.get_url(url, query = nil)
+    conn = Faraday.new(url: 'http://localhost:3000') do |faraday|
+      faraday.params[:query] = query unless query.nil?
+    end
+
+    response = conn.get(url)
+
+    data = JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
It looks like this method is exactly the same in `MerchantService`. It might be interesting to create another service class with a more generic name (ie `RailsEngineService`) where this method could live. That way, we could DRY up this code by making `ItemService` and `MerchantService` inherit from `RailsEngineService`